src/_gettsim/transfers/kinderzuschl/kost_unterk.py

```diff
@@ -21,71 +20,131 @@ def kinderzuschl_kost_unterk_m_bg(
     -------
 
     """
-    warmmiete_m_hh = bruttokaltmiete_m_hh + heizkosten_m_hh
-    anteil_warmmiete_m_bg = warmmiete_m_hh * _anteil_personen_in_haushalt_bg
 
-    out = _kinderzuschl_wohnbedarf_eltern_anteil_bg * anteil_warmmiete_m_bg
+    return (
+        bruttokaltmiete_m_bg + heizkosten_m_bg
+    ) * _kinderzuschl_wohnbedarf_eltern_anteil_bg
 
-    return out
 
-
-def bruttokaltmiete_m_bg(
+def bruttokaltmiete_m_bg(  # noqa: PLR0913
```

Comment:
Wouldn't a simpler solution be:
- calculate `bedarf_kosten_der_unterkunft_m` on the individual level (if part of a couple in the same hh, divide `kinderzuschl_params["existenzminimum"]["kosten_der_unterkunft"]["paare"]` by two
- use aggregates of this variable on the hh and bg level to determine `bruttokaltmiete_m_bg` etc.

Then we wouldn't need the variable `anz_paare_hh`

Reply:
Yes, maybe that's more useful in case we want to compute `bruttokaltmiete_m_eg` at some point.

We still need to determine whether a couple lives together in the same household, but we don't need the aggregation step of `anz_paare_hh`.

Reply:
yes, we probably need a variable `einstandspartner_in_household` or so which is non-standard. Not sure how difficult that is to generate.

Reply:
Shouldn't be a problem because this is part of `anz_paare_hh`